### DISCUSSION TITLE
Making Mstar/Touch mode

### DISCUSSION THREAD

**Discussion by gigawave07** [ppy/osu#15755] [Ideas] [answered]
I really like the Mstar/Touch game mode as show in this video https://www.youtube.com/watch?v=GK_4eqbCMEQ
But the current game content itself is limited by the publisher, it lacks all of the good feature in osu like creating / downloading map from community, no distracting avatar dancing, no leaderboards etc.
Thoughts on implement this game mode on osu?

**Comment by peppy** [comment, marked as answer]
We aren't developing any first-party rulesets for lazer for the foreseeable future, if ever. You might find a developer in the community interested though (which means you will be able to play it on osu!).
